src/scss/components/_advisory-panel.scss

```diff
@@ -1,6 +1,7 @@
 // CSS classes for the "Advisory Panel" page
 .advisory-panel {
 	.api-content > .wp-block-group {
```

Comment:
Only add the white background to the bio cards instead of having all the content in `.api-content` have a white background.

Reply:
> Only add the white background to the bio cards instead of having all the content in `.api-content` have a white background.

Good catch. Fixed and ready for another look.

Reply:
Looks good. After making that change it seems like re-adding the special css handling for the galaxy fold screen will work as intended now or we can leave it out and continue with relying on the soft hyphen solution.

Reply:
I lean towards relying on the soft hyphen solution which applies to all cases. It'll reduce the maintenance work by getting rid of special handlings.